.gitignore

```diff
@@ -29,3 +29,5 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+
+.env
```

Comment:
I'd suggest adding a `.env.sample` to the repository with the placeholder for the necessary ENV so that other developers see that this is needed.

Reply:
Added `.env.sample` file to the root of the project and reference in README file to it 👍